.gitignore

```diff
@@ -4,3 +4,5 @@ node_modules/
 npm-debug.log
 .eslint-release-info.json
 .nyc_output
+/yarn.lock
+/package-lock.json
```

Comment:
https://github.com/eslint/eslint-plugin-markdown/blob/610cffd93d29cc10bf9f2b51daeecb1b42338a34/.npmrc#L1

`package-lock.json` is disabled in `.npmrc`, so npm shouldn't be creating it. If someone does accidentally end up with a lock file (for either npm or yarn), I'd like that to show up in `git status` so that they notice it. Ensuring all installs use the latest compatible version will avoid one possible cause of weird hard-to-reproduce errors due to slightly different versions.

```suggestion
```

Reply:
It makes a bit difficult to add commits, I'm using `git add .` sometimes, so it work commit `yarn.lock` file accidentally. Did you mean that I should remove `yarn.lock` every time after installing? Or can I add a `.yarnrc` like `.npmrc`?

Reply:
Repositories under the ESLint GitHub organization have historically used npm in development and to run tests and releases. That means we've already solved this friction point for npm by automatically disabling `package-lock.json`, and I suspect that we'd set a similar option for yarn if one existed. I typically use yarn for other projects but use npm when working on ESLint repositories out of habit so hadn't noticed the `git add .` pain point until you just pointed it out.

I just checked a couple other repositories in the organization and see that they've included [both package managers' lockfiles in `.gitignore`](
https://github.com/eslint/eslint/blob/cd2e584aa5a87c425177fb84a9cd6b05624fec43/.gitignore#L24-L25). Given that precedent, if you'd like to submit a standalone PR that adds those, I'd be happy to merge it 🙂

Reply:
> I suspect that we'd set a similar option for yarn if one existed

Unfortunately, there is and will be no option for yarn.

I'd like to raise a PR for it.